Ports/aclock/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=aclock
+version=2.3
+# the github repo contains way too many different ports for other platforms and binaries, only fetch the file we need
```

Comment:
I don't think we even need the comment here, just downloading the file by itself should be pretty clear.